myrestapi/client/booksIndex.js

```diff
@@ -0,0 +1,8 @@
+async function showBooks(){
+    const URL= "http://localhost:8080/books";
+    const response = await fetch(URL);
+    console.log(response);
```

Comment:
Nit: You generally don't want to check in code with consoles logs. This is for a couple of reasons:

1. You don't want consoles going to production except for very special cases.
2. It's kind of a hygiene thing. When someone starts looking at the code they're gonna want to add their own console statements, if everyone checks these statements in the console will become unreadable with a bunch of random messages.